README.rst

```diff
@@ -157,34 +175,57 @@ Otherwise you can pull the trunk source and install from there::
 
     git clone git://github.com/bashwork/pymodbus.git
     cd pymodbus
+    
+To get latest release::
```

Comment:
Let's be explicit and say for now the latest release means `2.x` pymobuds with python 2 support.

Reply:
Done.